# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Add support for ru_RU.CP1251
# DP: Related bugs: #225516
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Petter Reinholdtsen
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10

Index: glibc-2.3.5/localedata/SUPPORTED
===================================================================
--- glibc-2.3.5.orig/localedata/SUPPORTED
+++ glibc-2.3.5/localedata/SUPPORTED
@@ -281,6 +281,7 @@
 ro_RO/ISO-8859-2 \
 ru_RU.KOI8-R/KOI8-R \
 ru_RU.UTF-8/UTF-8 \
+ru_RU.CP1251/CP1251 \
 ru_RU/ISO-8859-5 \
 ru_UA.UTF-8/UTF-8 \
 ru_UA/KOI8-U \
